src/test/java/org/openmrs/contrib/qaframework/automation/RolesAndPrivilegesSteps.java

```diff
@@ -0,0 +1,118 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AddEditNewPrivilegePage;
+import org.openmrs.contrib.qaframework.page.AddEditNewRolePage;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+import org.openmrs.contrib.qaframework.page.ManagePrivilegesPage;
+import org.openmrs.contrib.qaframework.page.ManageRolesPage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
```

Comment:
isn't  the import package `io.*` supposed to come first before `org.*`?

Reply:
i will do that. you meant, just changing location?, right? switching them up?